src/index.html

```diff
@@ -11,17 +11,410 @@
       rel="preconnect"
       href="https://fonts.gstatic.com"
     />
+    <link />
```

Comment:
It seems like you have an empty link tag on line 14. If it is not used, consider removing it to keep the code clean.